### ISSUE TITLE
Bug: Popover control should have aria attributes for accessibility

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-react#791]
### Description
The popover control is not being announced as such by screen readers.
To fix this, the popover control (the trigger) should have following aria attributes:
* [aria-controls](https://www.w3.org/TR/wai-aria-1.1/#aria-controls)
* [aria-expanded](https://www.w3.org/TR/wai-aria-1.1/#aria-expanded)
* [aria-haspopup](https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup)

VoiceOver on Chrome just announces 'button' without indication if it has an associated popover or not and if that popover is open or close:
![ACC-1424_no_aria](https://user-images.githubusercontent.com/43764832/68815177-8f8f8a80-062f-11ea-9294-a49d94024170.gif)

### Versions
**fundamental-react:** Current
**fundamental-styles:** N/A

**Comment by prsdthkr** [comment]
Fixed by https://github.com/SAP/fundamental-react/pull/796
